addons/web/static/src/search/search_model.js

```diff
@@ -1141,6 +1141,89 @@ export class SearchModel extends EventBus {
     // Private methods
     //--------------------------------------------------------------------------
 
+    /**
+     * Because it require a RPC call to get the properties search views items,
+     * it's done lazily, only when we need them.
+     */
+    async fillSearchViewItemsProperty() {
+        if (!this.searchViewFields) {
+            return;
+        }
+
+        const fields = Object.values(this.searchViewFields);
+
+        for (const field of fields) {
+            if (field.type !== "properties") {
+                continue;
+            }
+
+            const definitionField = fields.find((f) => f.name === field.definition_record);
+            const definitionRecordModel = definitionField.relation;
+            const definitionRecordField = field.definition_record_field;
+
+            const result = await this._fetchPropertiesDefinition(
```

Comment:
a domain should be added: no need for records with definitions = [].
Maybe also refactor the code of _fetchPropertiesDefinition using the new field service